### ISSUE TITLE
NVDA does not activate some links in IE 11

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5399]
**Reported by andrewd on 2015-10-06 02:30**
On a number of sites NVDA does not activate a link in IE but does so in Firefox.  It reports the link, but the new page will not open.  For example:on the page http://www.w3schools.com/js/js_if_else.asp the links to exercises cannot be activated.  The code for the link to Exercise 1 is <a class="w3-btn" href="exercise.asp?filename=exercise_conditions1" target="_blank">Exercise 1 &raquo;</a>.

Andrew

**Blocking #5402**

**Comment by ehollig** [comment]
I am able to open [Exercise 1](https://www.w3schools.com/js/exercise.asp?filename=exercise_conditions1) on [W3Schools](http://www.w3schools.com/js/js_if_else.asp) using IE 11 and latest NVDA. Can this be closed?

**Comment by jcsteh** [comment]
Closing as worksforme. Please reopen if the problem persists.
